index.qmd

```diff
@@ -1,55 +1,224 @@
+      Modular & composable
+    </div>
+    Turing is modular, written entirely in Julia, and is interoperable with the powerful Julia ecosystem.
+  </div>
+</div>
+```
 
-### Intuitive
-Turing models are easy to write and communicate — syntax is close to mathematical notations. 
+<div class="d-flex flex-row flex-wrap panel-wrapper gap-3 pb-2">
+<div class="example-text" style="text-align:right;padding:0.5rem;">
+  <div class="fs-4 fw-bold pb-1">
+  Hello, World in Turing
+  </div>
+  Some text about how easy it is to [get going](https://turinglang.org/docs/tutorials/00-introduction/).
+</div>
+<div class="example-code">
+```julia
+@model function coinflip(; N::Int)
+    # Prior belief about the probability of heads
+    p ~ Beta(1, 1)
 
-### General-purpose
-Turing supports models with discrete parameters and stochastic control flow. 
+    # Heads or tails of a coin are drawn from `N`
+    # Bernoulli distributions with success rate `p`
+    y ~ filldist(Bernoulli(p), N)
 
-### Modular and composable 
-Turing is modular, written entirely in Julia, and is interoperable with the powerful Julia ecosystem. 
+    return y
+end;
+```
+</div>
+</div>
+<div class="d-flex flex-row-reverse flex-wrap panel-wrapper gap-3 pt-2 section-end-space">
+<div class="example-text" style="padding:0.5rem;">
+  <div class="fs-4 fw-bold pb-1">
+  Goodbye, World in Turing
+  </div>
+  Some text about how easy it is to interface with external packages like AbstractGPs. Learn more about modelling [Gaussian Processes](https://turinglang.org/docs/tutorials/15-gaussian-processes/) with Turing.jl.
+</div>
+<div class="example-code">
+```julia
+@model function putting_model(d, n; jitter=1e-4)
+    v ~ Gamma(2, 1)
+    l ~ Gamma(4, 1)
+    f = GP(v * with_lengthscale(SEKernel(), l))
+    f_latent ~ f(d, jitter)
+    binomials = Binomial.(n, logistic.(f_latent))
+    y ~ product_distribution(binomials)
+    return (fx=f(d, jitter), f_latent=f_latent, y=y)
+end
+```
+</div>
+</div>
 
+```{=html}
+<h3 class="pb-3">
+  News
+</h3>
+```
+::: {#news}
 
-::: {.hero-buttons style="position: relative; display: flex; justify-content: center;"}
-[Get Started]({{< meta get-started >}}){.btn-action-primary .btn-action .btn .btn-lg role="button" style="background-color: #073c44; color: white"}
 :::
 
-::: {.g-col-3}
-:::
-:::
-:::
+[See all news &rarr;](news/)
 
-:::
+```{=html}
+<h3 class="pb-3 section-start-space">
+  Learn more
+</h3>
+<div class="list grid quarto-listing-cols-3">
+  <a class="pseudolisting" href="/library">
+    <div class="card-title h5">
+      Ecosystem
+    </div>
+    <p class="quarto-grid-item">
+      Learn about all the packages that are part of The Turing Organization.
+    </p>
+  </a>
+  <a class="pseudolisting" href="https://turinglang.org/docs/tutorials/docs-00-getting-started/">
+    <div class="card-title h5">
+      Documentation
+    </div>
+    <p class="quarto-grid-item">
+      Familiarize yourself with the Turing.jl API.
```

Comment:
```suggestion
      Familiarize yourself with the mechanics of Turing.jl.
```

API docs aren't on the docs website, they're on the individual repos (e.g. https://turinglang.org/Turing.jl). (Though this does makes me think we should at least mention this on the docs website)